core/src/main/resources/db/migration/V13__add_vehicle_blocked_install.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE Vehicle
+ADD blocked_install BOOL NOT NULL
+;
```

Comment:
this table is gone after #418 so we need to find a different solution for this pr.